cleaners/vuze.xml

```diff
@@ -82,8 +86,8 @@
     <action command="delete" search="glob" path="$$profile$$\torrents\*.tmp._az"/>
     <action command="delete" search="glob" path="$$profile$$\torrents\*.torrent._az.saving"/>
     <action command="delete" search="walk.all" path="$$profile$$\updates\"/>
-    <action command="delete" search="walk.all" path="%LocalAppData%\Temp\AzureusInstall\"/>
-    <action command="delete" search="glob" path="%LocalAppData%\Temp\AzureusInstall\"/>
-    <action command="delete" search="glob" path="%LocalAppData%\Temp\core_Azureus*"/>
+    <action command="delete" search="walk.all" path="%TMP%\AzureusInstall\"/>
+    <action command="delete" search="glob" path="%TMP%\AzureusInstall\"/>
+    <action command="delete" search="glob" path="%TMP%\core_Azureus*"/>
```

Comment:
The environment variable change looks fine. On line 90 search=glob should be `search="file"` because there is no wildcard in `path`

Reply:
I changed that!